src/lcm/create_params.py

```diff
@@ -119,9 +119,8 @@ def _create_stochastic_transition_params(
     """
     stochastic_variables = variable_info.query("is_stochastic").index.tolist()
 
-    # ----------------------------------------------------------------------------------
     # Assert that all stochastic variables are discrete state variables
-    # ----------------------------------------------------------------------------------
+    # ==================================================================================
```

Comment:
I am not a huge fan of these things, but fine with them if they won't ever change again... Can you do a bulk search & replace, please? *(maybe this is exactly what happened here, then just disregard)*